Add minutes for 2020-10-29 (#237)
srobo/competition-team-minutes: SR2021/2020-10-29.md

- We could in theory keep them open until the first deadline, but lets keep them officially open until a couple days before kickstart.
- Close on 19th (kickstart is on 21st)
- Give out team accounts on demand of sign-ups.
- [AI Jake Howard] draft the email to send to team leaders when their team is confirmed
## Recurring tasks:
- It’s a good idea to create tasks based on the recurring tasks repo
- [AI Peter Law ]: create all the tasks based on the recurring tasks repository (removing things unrelated to a virtual competition.
## Simulator:
- [AI Andy] Try the explosion mitigations (listed in slack by Andy B-S)
- [AI Jenny]  Check the segfault issue was fixed.
## Getting new volunteers in:
- [AI Jake Howard] Send an email to the volunteer mailing list telling them what we’re doing and how we need help
- We have a lot of tasks to be done, and so getting new volunteers in would be great.
## Game
- Alistair should have formed ideas into a rulebook (have bludgeoned him in slack)
## Handling external comms
- Jake to own team communication
- James to own public coms
  - [AI James Seden Smith] email trustees for social media access
- Andy to help with social media

# Actions

> inlined

Reviewer: Suggest duplicating them down here so we can have issues out of them.

> # Competition Team Meeting 2020-10-15

Reviewer 2: ```suggestion
# Competition Team Meeting 2020-10-29
```

## Attendees
- Jenny Fletcher
- Ash Holland
- Andy Barrett-Sprot
- James Seden-Smith
- Jake Howard
- Peter Law

## Topics

> ### Team signup

Reviewer: Empty line between headings please
```suggestion

### Team signup
```

- We’ve not sent out a form yet, but it will be done this week.
- [AI Jake Howard] send the email to teams to notify them of the event occurring
- Then we can add a sign-up form and send it out in an email (in a form that can be forwarded to other teams).
- [AI James Seden Smith] update sign-up form for 2021, ask for demographic information (rough numbers).

> ### Number of teams per organisation?

Reviewer: Empty line between headings please:
```suggestion

### Number of teams per organisation?
```

- Ask them how many they want and discuss with them if it’s more than 3 (to avoid a final league of people in the same organisation)
- Recommend team size of 4 (maybe 3?)

> ## When can we close entries?

Reviewer: Empty line between headings & keep them at a suitable level please:
```suggestion

### When can we close entries?
```

- We could in theory keep them open until the first deadline, but lets keep them officially open until a couple days before kickstart.
- Close on 19th (kickstart is on 21st)
- Give out team accounts on demand of sign-ups.
- [AI Jake Howard] draft the email to send to team leaders when their team is confirmed

> ## Recurring tasks:

Reviewer: Empty line between headings & keep them at a suitable level please:
```suggestion

### Recurring tasks:
```

- It’s a good idea to create tasks based on the recurring tasks repo
- [AI Peter Law ]: create all the tasks based on the recurring tasks repository (removing things unrelated to a virtual competition.

> ## Simulator:

Reviewer: Empty line between headings & keep them at a suitable level please:
```suggestion

### Simulator:
```

- [AI Andy] Try the explosion mitigations (listed in slack by Andy B-S)
- [AI Jenny]  Check the segfault issue was fixed.

> ## Getting new volunteers in:

Reviewer: Empty line between headings & keep them at a suitable level please:
```suggestion

### Getting new volunteers in:
```

- [AI Jake Howard] Send an email to the volunteer mailing list telling them what we’re doing and how we need help
- We have a lot of tasks to be done, and so getting new volunteers in would be great.

> ## Game

Reviewer: Empty line between headings & keep them at a suitable level please:
```suggestion

### Game
```

- Alistair should have formed ideas into a rulebook (have bludgeoned him in slack)

> ## Handling external comms

Reviewer: Empty line between headings & keep them at a suitable level please:
```suggestion

### Handling external comms
```

- Jake to own team communication
- James to own public coms
  - [AI James Seden Smith] email trustees for social media access
- Andy to help with social media

> # Actions
> - [AI Jake Howard] send the email to teams to notify them of the event occurring
> - [AI James Seden Smith] update sign-up form for 2021, ask for demographic information (rough numbers).
> - [AI Jake Howard] draft the email to send to team leaders when their team is confirmed
> - [AI Peter Law]: create all the tasks based on the recurring tasks repository (removing things unrelated to a virtual competition.
> - [AI Andy] Try the explosion mitigations (listed in slack by Andy B-S)
> - [AI Jenny]  Check the segfault issue was fixed.
> - [AI Jake Howard] Send an email to the volunteer mailing list telling them what we’re doing and how we need help
> - [AI James Seden Smith] email trustees for social media access

Reviewer: Suggest reformatting so these can be picked up by the actions tool:
```suggestion
## Action Points
- Jake Howard: send the email to teams to notify them of the event occurring
- James Seden Smith: update sign-up form for 2021, ask for demographic information (rough numbers).
- Jake Howard: draft the email to send to team leaders when their team is confirmed
- Peter Law: create all the tasks based on the recurring tasks repository (removing things unrelated to a virtual competition) (done).
- Andy: Try the explosion mitigations (listed in slack by Andy B-S)
- Jenny:  Check the segfault issue was fixed.
- Jake Howard: Send an email to the volunteer mailing list telling them what we’re doing and how we need help
- James Seden Smith: email trustees for social media access
```
Though you might also need to update the `.name-map.json` for some of these names.

> 
> 

Reviewer: Can we lose this trailing whitespace?
```suggestion
```
